save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/NewDemoToolRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.saveourtool.save.demo
+
+/**
+ * @property organizationName
+ * @property projectName
+ * @property vcsTagName
```

Comment:
could you please provide doc for this field, at least

Reply:
oh sure

Reply:
Done